docs/src/devdocs/AMR.md

```diff
@@ -77,7 +77,8 @@ julia> Ferrite._compute_size(3,1)
 Construct all level 1 octants based on mortonid:
 ```julia
 # note the arguments are dim,level,mortonid,maximumlevel
-julia> oct = OctantBWG(2,1,1,3)
+julia> dim = 2; level = 1; maximumlevel=3
```

Comment:
```suggestion
julia> dim = 2; level = 1; maximumlevel = 3
```

doing the important stuff here

Reply:
![grafik](https://github.com/Ferrite-FEM/Ferrite.jl/assets/27497290/2561c78f-e21f-4477-9341-f8baf6b5284c)